lib/plugins/aws/deploy/lib/updateStack.js

```diff
@@ -2,26 +2,24 @@
 
 const BbPromise = require('bluebird');
 const async = require('async');
-const merge = require('lodash').merge;
-const find = require('lodash').find;
-const forEach = require('lodash').forEach;
+const _ = require('lodash');
```

Comment:
Way better than requiring everything on it's own (don't know why I've done this before 😆 )!